packages/dev/docs/pages/releases/2023-10-02.mdx

```diff
@@ -0,0 +1,93 @@
+{/* Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we've added support for native links across all collection components in React Spectrum and React Aria! Now you can add links to components such as Menu, Breadcrumbs, Tabs and more. To streamline integration with client-side routers, we've introduced the RouterProvider component, allowing you to seamlessly incorporate client-side routers. See the client-side routing guide to learn how it set it up in your app. Since its beta release, we have made a number of improvements and bug fixes to React Aria Components. We have also added a new advanced customization guide that describes how to use contexts and hooks to build your own custom patterns using React Aria Components. React Spectrum has new additions, including DropZone and isPending prop to Button.
+
+date: 2023-10-02
+---
+
+# October 02, 2023 Release
+
+In this release, we've added support for native links across all collection components in React Spectrum and React Aria! Now you can add links to components such as `Menu`, `Breadcrumbs`, `Tabs` and more. To streamline integration with client-side routers, we've introduced the `RouterProvider` component, allowing you to seamlessly incorporate client-side routers. See the client-side [routing guide](../react-aria/routing.html) to learn how it set it up in your app.
+
+Since its beta release, we have made a number of improvements and bug fixes to [React Aria Components](../react-aria/react-aria-components.html). We have also added a new [advanced customization guide](../react-aria/advanced.html) that describes how to use contexts and hooks to build your own custom patterns using React Aria Components.
+
+For React Spectrum we have some new additions, including [DropZone](../react-spectrum/DropZone.html), static color support for [ProgressCircle](../react-spectrum/ProgressBar.html#static-color) and [ProgressBar](..react-spectrum/ProgressCircle.html#static-color), and support for [isPending](../react-spectrum/Button.html#pending) in `Button`.
```

Comment:
The ProgressCircle link here points to ProgressBar and the ProgressBar link is broken.

Reply:
```suggestion
For React Spectrum we have some new additions, including [DropZone](../react-spectrum/DropZone.html), static color support for [ProgressCircle](../react-spectrum/ProgressCircle.html#static-color) and [ProgressBar](../react-spectrum/ProgressBar.html#static-color), and support for [isPending](../react-spectrum/Button.html#pending) in `Button`.
```